airbyte-ci/connectors/pipelines/pipelines/telemetry.py

```diff
@@ -0,0 +1,42 @@
+import getpass
+import os
+import sys
+
+import segment.analytics as analytics
+
+analytics.write_key = "G6G7whgro81g9xM00kN2buclGKvcOjFd"
```

Comment:
We cool checking this in?

Reply:
I imagine so. We already have checked in the write key for octavia cli. I dont see the difference here.

Also not too disimilar to how sentry/segment FE libraries work